<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>10</y>
      <width>441</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QPushButton" name="createProfile">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="createName"/>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="updateName"/>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="createAge"/>
     </item>
     <item row="4" column="2">
      <widget class="QTextEdit" name="updateGender"/>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="updateProfile">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="updateID"/>
     </item>
     <item row="1" column="3">
      <widget class="QTextEdit" name="deleteID"/>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="updateAge"/>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="createGender"/>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="deleteProfile">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Id to be Updated/ Deleted</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startScan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start scan</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputLog">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>441</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>441</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QTextEdit" name="loginText"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Login to profile using Id:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
